README.md

```diff
@@ -93,7 +93,7 @@ Have a look at the code in the [examples](./examples) folder: with __master__, _
 * `init(options)`
 * `connect(options, connect_cb)`
 * `close()`
-* `path = await create(path, data, flags)`
+* `path = await create(path, data, flags, ttl)`
```

Comment:
I think it is a good idea to have an API with separate `create` functions (the "regular" and the "ttl" one). I browsed the [C client code](https://github.com/apache/zookeeper/blob/master/zookeeper-client/zookeeper-client-c/src/zookeeper.c#L4176) and there are two methods there, and it handles the `ttl` feature internally.

I think it is a good idea, if Apache changes something internally or discover backward compatibility issues. 

I understand it probably will cause some duplications in the `node_zk.cpp` of this library, but I think that is okay. What do you think?

Reply:
My personal opinion for this is that 
A) I think this way it's consistent with the API style as it is, where you already have a single call where optional arguments influence which features will be used
B) I would prefer less duplication as well as smaller API surface over many other things. Which I think is more easily achieved with this 'optional arguments' style, on the side of application code using the library as well.
Also, whether or not to have 2 separate create-functions on the native side only affects the interface between your native code and JS code, I don't really see how interface changes on the Zookeeper library side would have any effect on this. And the native side and JS side of this library always come together, so there wouldn't be any real issue in making a 'breaking' change there if that ever would be needed, right? (except when people in some weird way update the library but don't get a new binary)

But that's my opinion. It's your library so you can organize it any way you think is best :) If you just think it would make for a nicer API then you should go for it (but then you'd have to ask yourself: what about all those other parameters which are also optional?).